### DISCUSSION TITLE
Open start in all apps not working

### DISCUSSION THREAD

**Discussion by XiaomiDE** [valinet/ExplorerPatcher#2568] [General]
After the update from Windows i got today (Version 10.0.22631 Build 22631) this funtion does not work any more.
Yes, ExplorerPatcher is up to date.

![ApplicationFrameHost_lUmwImnwsH](https://github.com/valinet/ExplorerPatcher/assets/99801452/d4c4046b-24e6-4998-9208-d6e22056856d)
![rundll32_2hJvU6ONOV](https://github.com/valinet/ExplorerPatcher/assets/99801452/31029b2f-568a-47dd-a165-1d92d1a8acbd)

Thanks for looking into this,

Greetings from Belgium.

**Comment by pyrates999** [comment]
Your windows build is incorrect.  Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.

Please include any other software you have installed that modifies the UI of windows.

I also remember the option **open start in all apps by default** worked by automatically clicking the all apps part of the start menu after you've opened the start menu.  Microsoft doesn't have any actual code that would cause it to start in the all apps menu when you click on the start menu.  Therefore this doesn't work all the time by design.

It might be safer to just remove that option unless there is code that can actually do this.

**Comment by XiaomiDE** [comment]
> I also remember the option **open start in all apps by default** worked by automatically clicking the all apps part of the start menu after you've opened the start menu. Microsoft doesn't have any actual code that would cause it to start in the all apps menu when you click on the start menu. Therefore this doesn't work all the time by design.

But it worked for me before, this is interesting since i literally changed nothing but the update to make the menu pop up at the left side again. Anything else is as it has been before.

Winver tells me 23H2 (Build 22631.2792)

![rundll32_rbC2Nicbzx](https://github.com/valinet/ExplorerPatcher/assets/99801452/2f9cefb2-9399-43ab-88fd-8889bb42dc40)
![rundll32_fxazZM1VWt](https://github.com/valinet/ExplorerPatcher/assets/99801452/fd5133e4-c572-48c0-8d95-5201c420204c)
![rundll32_TOARha0Blb](https://github.com/valinet/ExplorerPatcher/assets/99801452/f48b8a12-039e-4790-a0e0-6bf26fa1b19b)
![rundll32_wl0OoOXNme](https://github.com/valinet/ExplorerPatcher/assets/99801452/72293660-c070-43a0-978e-4633802a2a5e)
![rundll32_L9kZ4GitYe](https://github.com/valinet/ExplorerPatcher/assets/99801452/7939e2bc-42fc-405a-8897-e7a54ed22eb1)
![rundll32_jcDInW8AFA](https://github.com/valinet/ExplorerPatcher/assets/99801452/69545951-f1be-44bb-9617-e160c41b5c5c)
![rundll32_peUBQPhUpb](https://github.com/valinet/ExplorerPatcher/assets/99801452/bdb10d7a-87a1-4d37-8ed7-296b3db07226)
![rundll32_iMFS9rZp2g](https://github.com/valinet/ExplorerPatcher/assets/99801452/b1a08c1b-363b-407d-87b7-876475c7903a)
![rundll32_aP5zhjJYdh](https://github.com/valinet/ExplorerPatcher/assets/99801452/b62297a6-8a88-46fb-99e1-8b6c4db4cf60)
![rundll32_5zzwaLI6LH](https://github.com/valinet/ExplorerPatcher/assets/99801452/efb698a9-7239-43ce-b030-26cac668f44f)



EDIT: Just treid around for a few times, switching around between win 11 and win 10 taskbar, followed by an explorer restart and now it's working again.
No idea why it bugged before. Will keep you posted if it messes up again.
